### DISCUSSION TITLE
What's the best way to write entries on the go on an Android smartphone?

### DISCUSSION THREAD

**Discussion by thymaro** [jendrikseipp/rednotebook#639] [Ideas]
I tested rednotebook on desktop and the app seems neat! I like Markdown and simple things in general.

I have a NAS at home and I also pay for a webhoster (although I might cancel that service in the future if I can figure out how to make my NAS come up with all the online services I wish for). With these prerequisites, how can I sync notes between desktop and mobile, so that I can make quick notes on the mobile device and do more elaborate stuff when writing at home with a full-fledged keyboard?

Would it be sensical to use Nextcloud synchronization, an SFTP instance, Joplin, some other Markdown editor? And how would you -- in general terms, I can look up the details afterwards -- go on about setting up such a system?

There are tons of note taking apps out there, but which one will save the files to a server of my choosing (i.e. my NAS or my host service) and in a format readable by rednotebook -- which is markdown, I know, and that's very good, but the file naming must be in a specific format, if I understood correctly. Is that so?

**Comment by jendrikseipp** [comment]
To be honest, if you require extensive synchronization between devices, I'd go for a program that explicitly supports desktop and mobile use. It's possible to synchronize RedNotebook journals, but there's currently no locking or merging mechanism: you need to make sure that you don't change both copies at the same time.

**Reply by thymaro** [reply]
That would be alright. I only have one other person who would need to have access. Hmm, maybe Etherpad would be a solution, but unfortunately, it doesn't support offline editing, if I'm not mistaken. Mmmahhh, it's hard.

**Reply by thymaro** [reply]
Thanks for the reply, anyways.
